spec/factories/uploads.rb

```diff
@@ -10,15 +10,11 @@
     is_consistent { true }
 
     trait :with_chunks do
-      chunks { [ build(:chunk, number: 1) ] }
+      chunks { [ create(:chunk, :skip_validation, number: 1) ] }
```

Comment:
I think this is a problem with the way we use :with_chunks.  Most of our test create(:upload, :with_chunks). If I pass upload: nil then all of these creates will fail because the resulting upload.chunks would be invalid (they need an upload). The few places were we build an upload, we should probably just build chunks explicitly.